GitUI/CommandsDialogs/RevisionDiffController.cs

```diff
@@ -117,13 +117,12 @@ public bool ShouldShowSubmoduleMenus(ContextMenuSelectionInfo selectionInfo)
 
         public bool ShouldShowMenuEditWorkingDirectoryFile(ContextMenuSelectionInfo selectionInfo)
         {
-            return selectionInfo.IsSingleGitItemSelected && !selectionInfo.IsAnySubmodule && selectionInfo.SingleFileExists;
+            return selectionInfo.IsSingleGitItemSelected && !selectionInfo.IsAnySubmodule && selectionInfo.IsAllFilesExists;
```

Comment:
This should also check that only one file is selected, add option isExactlyOneItemSelected
But remove ```selectionInfo.IsSingleGitItemSelected && !selectionInfo.IsAnySubmodule && ``` should not have been there before
```return selectionInfo.isExactlyOneItemSelected && selectionInfo.IsAllFilesExists;```

Reply:
@gerhardol I remove the ``IsAnySubmodule`.

``isExactlyOneItemSelected`` is the same than ``IsSingleGitItemSelected``.